packages/microcosm/src/action.js

```diff
@@ -264,10 +262,18 @@ class Action extends Emitter {
    * from history.
    */
   remove() {
+    console.assert(this.connected, 'Action has already been removed.')
+
+    this.connected = false
+
+    this.children.length = 0
```

Comment:
javascript please

Reply:
is `this.children = []` just as performant? because i think it reads wayyyy better

Reply:
Good question. It'll definitely allocate more memory. Basically I just want to purge children before an action goes away. Maybe we don't need to do this.